docs/getting-started/olmv1_getting_started.md

```diff
@@ -73,12 +73,11 @@ kubectl apply -f https://raw.githubusercontent.com/operator-framework/operator-c
 To upgrade the installed extension, update the version field in the ClusterExtension resource. Note that
 there must be CRD compatibility between the versions being upgraded, and the target version must be
 compatible with OLM v1. More information on CRD upgrade safety can be found [here](../concepts/crd-upgrade-safety.md),
-compatible with OLM v1. More information on CRD upgrade safety can be found [here](../concepts/crd-upgrade-safety.md),
-and on the extension upgrade process [here](../tutorials/upgrade-extension.md).
+and more information on the extension upgrade process can be found [here](../tutorials/upgrade-extension.md).
 
 ```bash
 # Update to v0.11.0
-kubectl patch clusterextension argocd --type='merge' -p '{"spec": {"source": {"catalog": {"version": "0.11.0"}}}}'
+kubectl patch clusterextension argocd --type='merge' -p '{"spec": {"source": {"catalog": {"version": "0.2.1"}}}}'
```

Comment:
Will this work?

I believe `https://raw.githubusercontent.com/operator-framework/operator-controller/main/config/samples/olm_v1_clusterextension.yaml` is currently installing `0.6.0`.

So if as a previous step we install `0.6.0` then here we effectively are trying to downgrade to `0.2.1`.

Reply:
Yeah, that's why I have my comment/question above about if we should change the sample manifest. The existing example of using 0.11.0 doesn't work either because it's attempting to upgrade a minor version within a zero major version which we don't support.

This specific line in 'olmv1_getting_started.md' technically isn't part of any tutorial or anything, but rather a one-off example of what an upgrade command might look like. So I think we could get away with whatever we want here, although having it be functional and not show an example of non-functional minor version upgrade attempts would probably be best. I could rewrite it to not have any actual version in the command, maybe, but rather some placeholder?

Reply:
@m1kola @everettraven so given we are adding a 0.2.0 manifest to the upgrade tutorial, should we just leave this line in olmv1_getting_started the same at 0.11.0 or maybe some other change?

Reply:
I don't like that we have duplication of content. Maybe we should consolidate the tutorial and getting started somehow?